### DISCUSSION TITLE
How do I disable the build in LSP snippets in LazyVim?

### DISCUSSION THREAD

**Discussion by jkbch** [LazyVim/LazyVim#363] [Q&A]
I want to disable the build in LSP snippets. 

I found this thread: https://github.com/hrsh7th/nvim-cmp/discussions/759

I put this in my lua/plugins/opts.lua but it does not work
`
  {
    "hrsh7th/cmp-nvim-lsp",
    opts = function()
      local capabilities = require("cmp_nvim_lsp").default_capabilities(vim.lsp.protocol.make_client_capabilities())
      capabilities.textDocument.completion.completionItem.snippetSupport = false
    end,
  },

`
Where would I put this in my Lazyvim config for it to work?

**Comment by ditsuke** [comment]
My question is along the same lines but more general, how do I enable/disable LSP capabilities?

Looking at the source code, the relevant section is https://github.com/LazyVim/LazyVim/blob/main/lua/lazyvim/plugins/lsp/init.lua#L85, which indicates we might be able to inject some options into the `lspconfig` opts to make that a part of the routine if thats not already possible.
